app/rag_google_module/file_search.py

```
@@ -45,10 +45,10 @@ async def _upload_catalog(self, jsonl_data: List[Dict]) -> genai.types.File:
 
         self.logger.info("Временный JSONL файл каталога создан. Загружаю...")
         try:
-            # Загружаем файл через aio namespace
+            # Загружаем файл через aio namespace с правильными параметрами
```

Ruff (0.14.4)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)